src/main/web/src/scenes/Search/components/SearchService.tsx

```diff
@@ -0,0 +1,158 @@
+import React, {useEffect, useMemo, useState} from "react";
+import "./SearchService.css";
+import Lottie from "lottie-react";
+import animationData from "../../../assets/loading.json";
+import {useDetectAdBlock} from "adblock-detect-react";
+import {usePreventScrolling} from "../../../organisms/ad-block-overlay/preventScrolling";
+import {getJWT} from "../../../services/AuthService";
+import config from "../../../config/config";
+import {PostModel} from "./PostModel";
+import {Post} from "./Post";
+import {User} from "./User";
+
+
+interface UserModel {
```

Comment:
Best practise: Extract this to a seperate file called `UserModel` in a folder `models`